recipe/meta.yaml

```diff
@@ -17,19 +17,21 @@ build:
 
 requirements:
   host:
+    - python >=3.7
+    - hatchling >=1.6.0,<2
     - pip
-    - python <4.0
   run:
-    - python <4.0
+    - python >=3.7
     # optional deps [secure]
-    - pyopenssl >=0.14
-    - cryptography >=1.3.4
+    - pyopenssl >=17.1.0
+    - cryptography >=1.9
     - idna >=2.0.0
     - certifi
+    - urllib3-secure-extra
```

Comment:
We don't want this as a dependency, if anything we should remove the optional deps for `[secure]` here (ie `pyopenssl`, `cryptography`, `idna`, `certifi`, etc)

Reply:
I don't recall why the secure optional deps were added but removing them now would probably cause some harm b/c folks do expected them now. We could do a multiple outputs and create a `urllib3-core` package and leave the `urllib3` with more "batteries."

Reply:
I agree this could cause disruption but this is a new major version? Disruption is expected, that's why we versioned this way. The [functionality that uses these dependencies is deprecated and will be removed from urllib3](https://github.com/urllib3/urllib3/issues/2680) before the end of 2023. Once the functionality is removed, then we can remove these dependencies?